functions/exercise2/script.js

```diff
@@ -7,4 +7,17 @@ function isPalindrome(phrase) {
     }
     var reversedNormalizedPhrase = normalizedPhrase.split("").reverse().join("");
     return reversedNormalizedPhrase === normalizedPhrase;
+}
+
+function showResult() {
+    var phrase = document.getElementById("phrase").value;
+    try {
+        var result = isPalindrome(phrase);
+        if (result) {result = "is a palindrome";}
+        else {result = "isn't a palindrome ";}
+        document.getElementById("result").value = `${phrase} ${result}`;
```

Comment:
We wanted to display phrase surrounded with single quotes ' to separate it from the rest of the message displayed.